domain/externalcontroller/state/types.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package state
+
+type ExternalController struct {
+	// Id holds external controller document key.
+	// ID is the controller UUID.
+	ID string `db:"uuid"`
```

Comment:
Should we have these if we're not currently using them? It might be confusing to others reading this.